guides/kubernetes.adoc

```diff
@@ -130,6 +130,56 @@ kubectl port-forward service/helloworld 8080:8080
 
 and access your application at http://localhost:8080/.
 
+## Deploy and expose the application to the outside world
+
+Now that you are satified with your application you might want to expose it to the outside world.
+The `wildfly` helm chart can alos create an `Ingress` resource for you so that your application is available oustide of the cluster.
```

Comment:
What are the prerequisite for this? How can I verify that my Kubernete has an Ingress controller? If not, we should link to some doc to let the user properly configure it (at least for Kind and Minikube)